### DISCUSSION TITLE
Install system-wide python

### DISCUSSION THREAD

**Discussion by pythoninja** [jdx/mise#1491] [Q&A] [answered]
Hi, is there any option available to install system-wide python with `mise`? Maybe I could pass additional parameters to the `python-build` script?

**Reference:** [Install Python of specific version system-wide with pyenv](https://stackoverflow.com/questions/41422826/install-python-of-specific-version-system-wide-with-pyenv)

**Comment by jdx** [comment, marked as answer]
https://github.com/jdx/mise/issues/1039

**Reply by pythoninja** [reply]
Thanks, any ETA for that feature?

**Reply by pythoninja** [reply]
@jdx, any ETA?

**Reply by jdx** [reply]
no, you're welcome to work on it though

**Reply by djmattyg007** [reply]
Unfortunately now that issues have been disabled, the answer to this question is unavailable :( can anyone shed light on what the issue described?

**Reply by pythoninja** [reply]
You can use the `install-into` option to install any package to the selected path.

https://mise.jdx.dev/cli/install-into.html
